### DISCUSSION TITLE
如何更改行内代码块样式

### DISCUSSION THREAD

**Discussion by NaughtyChas** [ppoffice/hexo-theme-icarus#1111] [Q&A] [answered]
默认的行内代码块样式（背景透明，字体颜色红色）感觉不太好看，有没有更改的方法？  

比如说Github这个样式就挺好看 -->  ```Example```

**Comment by ppoffice** [comment, marked as answer]
@NaughtyChas 可以在`source/css/default.styl`定义想要的code样式。参考https://github.com/groenroos/bulma-stylus/blob/v0.8.0/stylus/base/generic.styl#L82-L87

**Reply by NaughtyChas** [reply]
十分感谢您的回复。  @ppoffice 

https://github.com/groenroos/bulma-stylus/blob/60687bc9680ac3d06071115687bc7cfa54272c3d/stylus/base/generic.styl#L82-L87  

修改Line84`code`的值可以使代码颜色改变，但是修改Line83`background-color`无法改变代码块背景颜色。  
会不会是修改了错误的变量？  

本人经验不足，求大佬指点:pray::pray:

**Reply by ppoffice** [reply]
https://github.com/ppoffice/hexo-theme-icarus/blob/master/include/style/article.styl#L75-L78

**Reply by NaughtyChas** [reply]
@ppoffice 感谢！🙏  

另外，可以自己添加HTML属性自定义盒子模型样式，如`box-shadow`，`border-color`等。   
对应属性及参数可直接添加在https://github.com/ppoffice/hexo-theme-icarus/blob/master/include/style/article.styl#L75-L78 高亮区域内。  

行间代码块样式可于`source/css/default.styl`处更改。  
需将https://github.com/groenroos/bulma-stylus/blob/v0.8.0/stylus/base/generic.styl 内容复制进`source/css/default.styl`文件，然后在[此处](https://github.com/groenroos/bulma-stylus/blob/60687bc9680ac3d06071115687bc7cfa54272c3d/stylus/base/generic.styl#L120-L133) 高亮区域内增加或改变属性及参数。
